docs/DeveloperGuide.md

```diff
@@ -551,6 +614,32 @@ MSS:
 
    Use case ends.
 
+**UC0X - Reminders for upcoming sessions**
+
+MSS:
+
+1. User toggles to the `Home` tab.
+2. TutorBuddy shows a list of upcoming sessions that would happen, within the next 3 days.
+
+   Use case ends.
+
+**UC0X - Calendar View**
+
+MSS:
+
+1. User toggles to the `Calendar` tab.
+2. TutorBuddy shows a calendar representation of the sessions, showing the schedule of the current week.
+
+   Use case ends.
+
+Extensions:
+
+* 2a. User can toggle between the different weeks using the left and right button in the Calendar page.
+
+* 2b. User can toggle directly move to this week's schedule by clicking on the `Today` tab.
```

Comment:
i think should be `Today` button instead.

Reply:
Ok nice catch! Have updated it accordingly.